draft-huitema-quic-in-space.md

```diff
@@ -78,6 +78,12 @@ network management considerations are out of scope of this document.
 
 {::boilerplate bcp14-tagged}
 
+# Path Changes
+
+In deep space in a direct communication scenario such as an Earth ground station to directly an asset on Mars, the RTT is large but pretty stable. However, when an orbiter acting as a communication relay is used, the RTT has a very different pattern. As a simple example scenario, Earth communicates to orbiter which communicates to a Mars asset and vice-versa, Mars is at its nearest distance to Earth (4 minutes delay) and the orbiter takes 2 hours to make a circular orbit, which means 1 hour reachable from Earth. In this scenario, the Earth ground station will see a stable RTT (~8 minutes) to the orbiter when the orbiter is directly reachable from Earth and the asset is reachable from the orbiter. When the orbiter is no more reachable as it is going on the far side, the ground station will accumulate packets over time, and then when the orbiter becomes reachable again, will send those buffered packets. The first buffered packets just after the orbiter is unreachable will have 1 hour + 8 minutes RTT. The last buffered packets just before the orbiter becomes reachable will have a 8 minutes RTT. In between, it will be essentially linear. In this simple scenario, the RTT shows a sawtooth pattern with some stable value period. Obviously, depending on the location of the asset, the type of orbit, the number of orbiters and assets, etc, that pattern will be more complex and different. However, it will still show an important jump, like from 8 minutes to 1 hour, in the RTT.
+
+The bandwidth of a path in deep space may change based on the conditions. However, experience with Mars (ref to study) shows that it is pretty constant for the same orbiter links to Earth and to the assets. However, if a different orbiter is used on the return path and/or if  optical links are used on some legs where radio links were used previously, then bandwidth would change. However, this is pretty theorical, as paths and routing is very static and planned in advance in space. Therefore, a bandwidth change in the path within a short period of time, or within the time of some packets exchange will be exceptional. If a connection is kept for weeks or months, then it is possible then a few times the path bandwidth will change.
+
```

Comment:
How far are we from inter-orbital relays?

Reply:
good question. in space very difficult to predict, since lunar landing missions as we have seen have been very difficult. >10 years maybe? Current plans from space agencies are even way further, but...

Reply:
I am proposing a change in text to make it open to multiple topologies.